### PR TITLE
Some Improvements

### PR DESCRIPTION
**Pull request by exweilt**
Increasing robot selection limit is something definitely needed
Creating a robot at the cursor is pretty useful when debugging game logic(at least for me)

### CODE REVIEW

File: MatrixGame/src/MatrixMultiSelection.cpp [hunk 1]
```diff
@@ -253,7 +253,7 @@ void CMultiSelection::Update(
                 SETFLAG(m_Flags, MS_FLAG_BUILDINGS);
             }
 
-            if (m_SelItems.size() < 9) {
+            if (m_SelItems.size() < 30) {
```
Comment (vladislavrv) [inline MatrixGame/src/MatrixMultiSelection.cpp:256]: please consider creating some constant (`constexpr` or `define` on your taste) for the magic number. right in this file i guess, doesn't make much sense to have it globally.

File: MatrixGame/src/MatrixDebugInfo.cpp [hunk 2]
```diff
@@ -21,8 +21,8 @@ CMatrixDebugInfo::CMatrixDebugInfo()
 : m_Font{nullptr}
 , m_Pos{DI_KEY_X, DI_KEY_Y}
 {
-    if (D3D_OK != D3DXCreateFontW(g_D3DD, 10, 0, FW_NORMAL, 1, FALSE, DEFAULT_CHARSET, OUT_DEFAULT_PRECIS,
-                                  DEFAULT_QUALITY, DEFAULT_PITCH, L"MS Sans Serif", &m_Font))
+    if (D3D_OK != D3DXCreateFontW(g_D3DD, 30, 0, FW_NORMAL, 1, FALSE, DEFAULT_CHARSET, OUT_DEFAULT_PRECIS,
```
Comment (vladislavrv) [inline MatrixGame/src/MatrixDebugInfo.cpp:24]: TBH i'm not sure if such a big text is necessary (on my FHD screen it looks HUGE), but i don't mind.  
just please make sure `DI_KEY_W` is big enough for the keys to not overlap with values at least for the cheats (e.g. try `KEEPALIVE` cheat)

**Review by vladislavrv — CHANGES_REQUESTED** [review]

**Comment by vladislavrv** [comment]
Hey, welcome to the ~~jungle~~ refactoring!  
Seems like github ignores sending me any notifications, so sorry for delay, here and later...
On the MR - couple minor comments, and i believe you've carefully tested it ;)

File: MatrixGame/src/MatrixMultiSelection.cpp (hunk 1 above)
Comment (exweilt) [reply MatrixGame/src/MatrixMultiSelection.cpp:256]: Makes sense.

File: MatrixGame/src/MatrixDebugInfo.cpp (hunk 2 above)
Comment (exweilt) [reply MatrixGame/src/MatrixDebugInfo.cpp:24]: I have FHD too, but it isn't big at all, like ~2-3% of window height(in exe) per row.
Anyway its matter of tastes
other point is valid

**Pull request closed without merging**